src/components/user/login.jsx

```diff
@@ -386,10 +386,10 @@ function Login() {
                     )}
 
                     {usernamePasswordLogins.length > 0 &&
-                      usernamePasswordLogins.map((loginMethod) => (
-                        <div key={loginMethod.provider}>
+                      usernamePasswordLogins.map(({ label, type }) => (
```

Comment:
Why the change from key=loginMethod.provider til loginMethod.type ?

Reply:
This will not result in unique keys

Reply:
Are the types not distinct? Because there were errors in the console stating a unique key was missing.